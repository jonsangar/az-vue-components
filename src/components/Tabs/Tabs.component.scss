/* AZ TABS
*  TODO: llevarse el CSS a un componente Tabs
*/

.az-tabs{
  @include shadow(3);

  position: relative;
  display: flex;
  width: 100%;
  margin-bottom: simple();
  overflow: hidden;
  background-color: #ffffff;
  border: none;
  border-radius: $border-radius;

  a{
    color: inherit;
  }

}
