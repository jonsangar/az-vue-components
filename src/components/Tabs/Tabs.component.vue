<template lang="pug">
  .az-tabs
    slot
</template>

<script>
export default {
  name: 'az-tabs'
}
</script>

<style lang="scss" src="./Tabs.component.scss" scoped></style>
