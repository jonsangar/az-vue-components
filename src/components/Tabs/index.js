import Tabs from './Tabs.component.vue';

export default Tabs;
