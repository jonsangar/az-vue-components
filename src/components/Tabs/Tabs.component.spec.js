import { shallowMount, createLocalVue } from '@vue/test-utils';
// Importamos librerÃ­as de estilos e iconos
import SvgIcon from 'vue-svgicon'
import Tabs from './Tabs.component.vue';

const localVue = createLocalVue()
localVue.use(SvgIcon, {
  tagName: 'az-icon',
  classPrefix: 'az',
})

describe('Tabs.vue', () => {
  const build = () => {
    const wrapper = shallowMount(Tabs, {
      localVue,
    })

    return {
      wrapper,
    }
  }

  it('Comprueba renderizado del componente Tabs con snapshot', () => {
    const { wrapper } = build()
    expect(wrapper.html()).toMatchSnapshot()
  })


});
