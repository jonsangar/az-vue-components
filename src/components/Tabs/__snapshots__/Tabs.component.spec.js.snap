// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tabs.vue Comprueba renderizado del componente Tabs con snapshot 1`] = `<div class="az-tabs"></div>`;
